packages/client/src/forms/register/fieldDefinitions/register.ts

```diff
@@ -9,162 +9,294 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
-import { IForm } from '@client/forms/index'
+import { RadioSize } from '@client/../../components/lib/forms'
+import {
+  /* IForm, */
+  IFormConfig,
+  BirthSection,
+  REVIEW_OVERRIDE_POSITION,
+  FLEX_DIRECTION,
+  ISerializedForm,
+  DeathSection,
+  IQuestionIdentifiers,
+  IQuestionConfig,
+  SerializedFormField,
+  ISerializedFormSection,
+  IFormSectionGroup,
+  QuestionConfigFieldType
+} from '@client/forms/index'
+import { conditionals } from '@client/forms/utils'
+import { formMessageDescriptors } from '@client/i18n/messages'
+import { set } from 'lodash'
+import { MessageDescriptor } from 'react-intl'
 
-/*interface IRegisterForms {
-  birth: IForm
-  death: IForm
-}*/
+interface IDefaultRegisterForms {
+  birth: ISerializedForm
+  death: ISerializedForm
```

Comment:
Eventually, I would love to be able to replace ISerializedForm with IForm and remove the need for this file: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/forms/mappings/deserializer.ts